### PR TITLE
Amsinh/fixgeneratorbug

### PR DESCRIPTION
**Pull request by ameyanator**

### CODE REVIEW

File: generators/ReliableStatefulService/index.js [hunk 1]
```diff
@@ -19,10 +19,10 @@ var ClassGenerator = generators.Base.extend({
     this.libPath = this.options.libPath;
     this.isAddNewService = this.options.isAddNewService;
   
-  },
+  }
   
-  prompting: function () {
-    var done = this.async();
+  async prompting() {
+    // var done = this.async();
```
Comment (vshan) [inline generators/ReliableStatefulService/index.js:25]: nit: remove unused lines. other places as well

File: generators/app/index.js [hunk 2]
```diff
@@ -35,16 +35,16 @@ var JavaGenerator = generators.Base.extend({
...
-    this.prompt(prompts, function (props) {
+    await this.prompt(prompts).then(props => {
       this.props = props;
       this.props.projName = this.props.projName.trim();
       this.config.set(props);
 
-      done();
-    }.bind(this));
-  },
+      // done();
```
Comment (vshan) [inline generators/app/index.js:43]: nit: unused line

File: package.json [hunk 3]
```diff
@@ -19,7 +19,7 @@
   "dependencies": {
     "chalk": "^3.0.0",
     "xml2js": "^0.4.23",
-    "yeoman-generator": "^4.5.0",
+    "yeoman-generator": "^5.4.0",
```
Comment (vshan) [inline package.json:22]: @ameyanator have you validated all the scenarios with this plugin, like create app/delete app/deploy/build etc?

File: generators/ReliableStatefulService/index.js (hunk 1 above)
Comment (ameyanator) [reply generators/ReliableStatefulService/index.js:25]: aah I'll remove them. my bad

File: generators/app/index.js (hunk 2 above)
Comment (ameyanator) [reply generators/app/index.js:43]: aah I'll remove them. my bad

File: package.json (hunk 3 above)
Comment (ameyanator) [reply package.json:22]: I validated create and delete. I'll check for deploy and build too

**Review by vshan — APPROVED** [review]

**Pull request merged**